mne/datasets/bnci/bnci.py

```diff
@@ -0,0 +1,608 @@
+# Author: Alexandre Barachant <alexandre.barachant@gmail.com>
+# License: BSD Style.
+
+import os
+from os import path as op
+import numpy as np
+
+from scipy.io import loadmat
```

Comment:
we use nested imports for scipy modules : ['integrate', 'special', 'optimize', 'io']

that's the cause of the travis failure.